docker/local.Dockerfile

```diff
@@ -37,6 +37,6 @@ ENV FM_ETH__LISTEN__HOST=0.0.0.0
 # We could build the actor bundles in the `builder` as well,
 # but we should be able to copy it from somewhere.
 COPY docker/.artifacts/bundle.car $FM_HOME_DIR/bundle.car
-COPY docker/.artifacts/contracts $FM_HOME_DIR/contracts
+COPY docker/.artifacts/contracts/out $FM_HOME_DIR/contracts
```

Comment:
I found a different fix for this, [here](https://github.com/consensus-shipyard/fendermint/pull/319/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R15). 

The problem was that the way `IPC_ACTORS_OUT` was constructed found a multitude of `out` dirs since `bindings` were added. But the way it's [used](https://github.com/consensus-shipyard/fendermint/blob/8a30ea017a2071ab234faedcc42f8df25eb05db2/Makefile#L66) is clear: we intend to copy the contents of the out directory itself. 

So I'd say this fix isn't needed, it's not the right fix, because `docker/.artifacts/contracts` should contain the contracts directly.

Reply:
Gotcha. I included your fix and it works. Thanks!